app/controllers/application_controller.rb

```diff
@@ -91,6 +93,19 @@ def swaddled
     response.headers["swaddled-by"] = "rubyforgood"
   end
 
+  def after_sign_in_path_for(resource)
```

Comment:
As someone unfamiliar with Devise, does this override the default that would take the user to the `resource_return_to`?